pkg/config/config_template.yaml

```diff
@@ -63,12 +63,15 @@ api_key:
 # sslkeylogfile: ""
 
 
-## @param force_tls_12 - boolean - optional - default: false
```

Comment:
Would it make sense to keep this option documented? (at the very least, because it also mentions env var name). If we say that this is equivalent to setting `min_tls_version: 1.2`, I think it would also simplify the docs for `min_tls_version`, which could then say that only `1.0` is the default.

Reply:
I'd probably agree with this since all functionality of this config option is covered by `min_tls_version` options.

Reply:
Good, since I already un-deprecated it :)